ctwrap/simulation.py

```diff
@@ -306,7 +405,16 @@ def tasks(self):
         e = self._entry
         return {'{}_{}'.format(e, v): v for v in self._values}
 
-    def run_task(self, sim, task, **kwargs):
+    def run_task(self, sim: Tsimulation, task: str, **kwargs:str):
```

Comment:
I believe you can use `Simulation` here. `Tsimulation` is only needed when a `@classmethod` returns an object?

Reply:
PS: what I found is that it's sufficient to use quotes when returning a new class instance, i.e. the `T` classes aren't really necessary.